app/controllers/admin/dashboard_controller.rb

```diff
@@ -5,4 +5,25 @@ def index
   def show_users
     @users = User.all
   end
+
+  def show_merchants
```

Comment:
I commented about this on the `routes.rb` file, but for ReSTful routes, should we remove `_[user]` from the method names...?

Reply:
that would be more restful but one of them would still have to be either show_users or show_merchants. I just thought since one would be unrestful it would be more helpful to us to name them more explicitly since there are basically two show actions. I'm curious about others' thoughts on this though.